Ginger/GingerCoreNET/WorkSpaceLib/Solution.cs

```diff
@@ -227,6 +228,83 @@
 
         private string mAccount;
 
+        /// <summary>
+        /// For encrypting password variables
+        /// </summary>
+        public string EncryptionKey { get; set; }
+
+        [IsSerializedForLocalRepository]
+        public string EncryptedSolutionName {get;set;}
+
+        public bool ValidateKey()
+        {
+            try
+            {
+                return EncryptionHandler.DecryptwithKey(EncryptedSolutionName, EncryptionKey).Equals("valid");
+            }
+            catch
```

Comment:
Add Debug level logs with Reporter.ToLog(eLoglevl.Debug)
Having empty catch is not a good practice

Reply:
Added logs in empty catches